src/Invoice.php

```diff
@@ -91,6 +134,17 @@ public function getTotals(): InvoiceTotals
         return $this->totals;
     }
 
+    public function getInvoiceAmount()
```

Comment:
Can you make this work with `Money\Money`?

Reply:
Fixed in #142